packages/tldraw/src/lib/utils/export/copyAs.ts

```diff
@@ -10,10 +10,10 @@ import {
 	doesClipboardSupportType,
 	getAdditionalClipboardWriteType,
 } from '../clipboard'
-import { exportToBlobPromise, exportToString } from './export'
+import { exportToImagePromise } from './export'
 
 /** @public */
-export type TLCopyType = 'svg' | 'png' | 'jpeg' | 'json'
+export type TLCopyType = 'svg' | 'png'
```

Comment:
so for `jpeg`, have you considered using `web image/jpeg`? related issue: https://github.com/tldraw/tldraw/issues/4873
to be clear, i haven't messed around with that new API to see if it works or not.

Reply:
the issue with `web image/jpeg` is that whilst it works for copying a jpeg back into tldraw, it won't interop with most other software unfortunately - unless they've explicitly added support for these custom formats. I think to most users in most use-cases, this would make the feature appear broken :(

Reply:
fair enough 👍 thought i'd check